telco-hub/install/openshift/install-config.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: v1
+metadata:
+  name: hub  # replace with your hub name
+baseDomain: example.com  # replace with your domain name
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.125.0/24  # replace with your machine network CIDR
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+# Replace the fields below with your network details
+platform:
+  baremetal:
+    provisioningNetwork: "Disabled"
+    apiVIPs:
+    - 192.168.125.10
+    ingressVIPs:
+    - 192.168.125.11
+    hosts:
+    - bootMACAddress: aa:aa:aa:aa:01:01
+      name: hub-ctl-0
+      role: master
+    - bootMACAddress: aa:aa:aa:aa:01:02
+      name: hub-ctl-1
+      role: master
+    - bootMACAddress: aa:aa:aa:aa:01:03
+      name: hub-ctl-2
+      role: master
+bootstrapInPlace:
+  installationDisk: /dev/vda
```

Comment:
Our recommendation is to use /dev/disk/by-path instead of the assigned device names. Can you verify that works in this context and replace with "by-path". In some cases the assigned device names can change based on how the kernel enumerates the devices, but the by-path is static based on the installed slot in the server.

Reply:
After looking a bit deeper into this I realized that this config is only valid for SNOs. For a compact cluster it seems to be ignored as I've verified that OCP is not installed in the disk pointed by this config. I have used instead the `platform.baremetal.hosts.rootDeviceHints.deviceName` field with a disk identified "by-path" and it works. I will update the PR.